src/site/content/en/blog/handwriting-recognition/index.md

```diff
@@ -0,0 +1,426 @@
+---
+title: Recognize your users' handwriting
+subhead:
+  The Handwriting Recognition API allows you to recognize text from handwritten input in real time.
+description: |
+  The Handwriting Recognition API allows web applications to make use of advanced handwriting recognition services to recognize text from hand drawings in real time.
+authors:
+  - christianliebel
+  - thomassteiner
+
+date: 2021-05-14
+# updated: 2021-05-14
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/k1gdvpBMneFVrOC5h4yQ.jpg
+alt: A hand draws letters in calligraphic script on a paper.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+
+origin_trial:
+  url: https://developer.chrome.com/origintrials/#/view_trial/🎉TODO_UPDATE🎉
+---
+
+{% Aside %} The Handwriting Recognition API is part of the [capabilities project](/fugu-status/) and
+is currently in development. This post will be updated as the implementation progresses.
+{% endAside %}
+
+## What is the Handwriting Recognition API? {: #what }
+
+> TODO: GIF of demo
+
+The Handwriting Recognition API is a new web platform API that allows you to convert handwritten
+drawings (ink) from your users into text. Some operating systems have long included such APIs,
+and with this new capability, your web apps can finally leverage the same powerful functionality.
+The conversion takes place directly on the user's device, works even in offline mode, all without
+adding any third-party libraries or services.
+
+This API implements a so-called "on-line" or real-time recognition. This means that the handwritten
+input is recognized while the user is drawing it, by capturing and analyzing the single strokes. In
+contrast to "off-line" procedures such as Optical Character Recognition (OCR), where only the end
+product is known, on-line algorithms can provide a higher level of accuracy.
+
+### Suggested use cases for the Handwriting Recognition API {: #use-cases }
+
+Examples of sites that may use this API include:
+
+- Note-taking applications where users want to capture handwritten notes and have them translated
+  into text.
+- Forms applications where users use pen or finger input due to time constraints.
+- Games that require filling in letters, such as crosswords or Hangman.
```

Comment:
```suggestion
- Games that require filling in letters, such as crosswords, sudoku or hangman.
```

Reply:
```suggestion
- Games that require filling in letters or numbers, such as crosswords, hangman, or sudoku.
```

So that letter/number examples are in order?